pipelines/resources/spokes/spoke-deploy-ui.yaml

```diff
@@ -21,6 +21,9 @@ spec:
     - name: mock
       type: string
       default: "false"
+    - name: ztpui-container-image
```

Comment:
We will need to add this parameter also in the `pipeline/resources/deploy-ztp-spokes.yaml`, this is an example of how more or less should looks like. (Maybe I missed something on the sample xD)

```
spec:
  description: Tekton Pipeline to deploy ZTPFW Spokes clusters
  params:
    - name: ztp-container-image
      type: string
      default: "quay.io/ztpfw/pipeline:latest"
    - name: ztpui-container-image
      type: string
      default: "quay.io/ztpfw/ui:latest"
...
...
  workspaces:
    - name: ztp
  tasks:
...
...
  - name: deploy-ui
    taskRef:
      name: spoke-deploy-ui
    params:
      - name: spokes-config
        value: $(params.spokes-config)
      - name: kubeconfig
        value: $(params.kubeconfig)
      - name: ztp-container-image
        value: $(params.ztp-container-image)
      - name: UI_IMAGE
        value: $(params.ztpui-container-image)
      - name: mock
        value: $(params.mock)
    runAfter:
      - deploy-icsp-spokes-post
    workspaces:
      - name: ztp
...
...
```